ergo-v0.1.0

EsotericSquishyy opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [x] a new package
- [ ] an update for a package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

Description: The package provides many customizable block environments that can be used to take notes in primarily Math, Computer Science, and Physics.

<!--
These things need to be checked for a new submission to be merged. If you're just submitting an update, you can delete the following section.
-->

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

elegaanz reviewed (commented):
Thank you for contributing this package, it looks nice! I have one small remark, but after that it should be good to be merged.

Edit: and also it would be nice to fix the kebab-case related warnings, as kebab-case is the most common naming convention in the Typst ecosystem (but if you have a strong preference for snake_case, we can also accept it as is).

EsotericSquishyy commented on packages/preview/ergo/0.1.0/typst.toml:
```diff
@@ -0,0 +1,12 @@
+description = "A suite of customizable block environments for taking notes or doing problem sets in Mathematics, Computer Science, Physics, and more."
+disciplines = ["mathematics", "computer-science", "physics"]
+license = "MIT-0"
+exclude = [ "gallery/*", "examples/*", "utils/*" ]
```
Would it be possible to keep the .typ files that are in examples. The intention was that the README could link the svgs to those .typ files to provide reference on typst universe.

EsotericSquishyy commented:
Let me know if these changes are sufficient:
- Changed public identifiers to kebab-case
- Removed examples directory and moved example typ files to the gallery directory (still excluded in toml)
- Updated README to use `@preview`

elegaanz commented:
That works, thanks!
